docs/book/src/topics/flannel.md

```diff
@@ -0,0 +1,68 @@
+# Flannel
```

Comment:
I wonder if there should be some mention of using Calico in here since that's what we use in tests and quickstart docs right now but there's no mention of it anywhere in this book so it might be confusing to users 🤔  What do you think about adding a short sentence about other CNIs in here and pointing to the Calico template and/or opening a follow up issue for adding more comprehensive docs on Azure + Calico?